app/views/spaces/show.html.erb

```diff
@@ -1,2 +1,15 @@
 <h1>Spaces#show</h1>
 <p>Find me in app/views/spaces/show.html.erb</p>
+
+<ul class="list-group">
+  <% spaces.each do |space| %>
+    <li class="list-group-item">
+      <%= space.location %>
+      <%= space.description %>
+      <%= space.volume %>
+      <%= space.price_per_month %>
```

Comment:
All of these should start with `@space` instead and be placed in their own list item element.